pyoozie/client.py

```diff
@@ -561,6 +561,25 @@ def jobs_submit_coordinator(self, xml_path, configuration=None):
             return coord
         raise exceptions.OozieException.operation_failed('submit coordinator')
 
+    def jobs_update_coordinator(self, coordinator_id, xml_path, configuration=None):
+        user = self._user or 'oozie'
+        coord = self._fetch_coordinator_or_action(coordinator_id)
+        if coord.status.is_active():
+            conf = xml._coordinator_submission_xml(user, xml_path, configuration=configuration)
+            if self._verbose:
+                self.logger.info('Preparing to update coordinator %s:\n%s', xml_path, conf)
+            reply = self._put('job/{}?action=update'.format(coordinator_id), conf)
+
+            if not reply and 'update' not in reply:
```

Comment:
If I try to update the coordinator with a new one that has a different frequency, the request throws a nice error:

If I changed the frequency the request throws an exception...That's actually helpful:
```
OozieCommunicationException: E0803: IO error, E1023: Coord Job update Error: [Frequency can't be changed. Old frequency = 140 new frequency = 1440]
```